bin/all_sky_search/pycbc_fit_sngls_over_multiparam

```diff
@@ -236,22 +241,133 @@ for inputstr in smooth_kwargs:
         key, value = inputstr.split(':')
         kwarg_dict[key] = value
     except ValueError:
-            err_txt = "--smoothing-keywords must take input in the " \
-                      "form KWARG1:VALUE1 KWARG2:VALUE2 KWARG3:VALUE3 ... " \
-                      "Received {}".format(' '.join(args.smoothing_keywords))
-            raise ValueError(err_txt)
+        err_txt = "--smoothing-keywords must take input in the " \
+                  "form KWARG1:VALUE1 KWARG2:VALUE2 KWARG3:VALUE3 ... " \
+                  "Received {}".format(' '.join(args.smoothing_keywords))
+        raise ValueError(err_txt)
 
 assert len(args.log_param) == len(args.fit_param) == len(args.smoothing_width)
 
 init_logging(args.verbose)
 
-fits = h5py.File(args.template_fit_file, 'r')
+analysis_time = 0
+attr_dict = {}
+
+# This will be used to check the fit files have the same number of templates
+# (therefore probably using the same bank)
+num_templates = None
+
+# These end up as n_files * n_templates arrays
+tid = numpy.array([], dtype=int)
+nabove = numpy.array([], dtype=int)
+ntotal = numpy.array([], dtype=int)
+alpha = numpy.array([], dtype=float)
+median_sigma = numpy.array([], dtype=float)
+
+logging.info("Loading input template fits")
+for filename in args.template_fit_file:
+    with h5py.File(filename, 'r') as fits:
+        if num_templates is None:
+            # First file; store number of templates
+            num_templates = fits['template_id'].size
+        elif not num_templates == fits['template_id'].size:
+            raise RuntimeError(
+                "Input fit files correspond to different banks. "
+                "This situation is not yet supported."
+            )
+        # get attributes from the template-level fit
+        for k in fits.attrs.keys():
+            if k == 'analysis_time':
+                # For this attribute only, we want the mean
+                analysis_time += fits.attrs['analysis_time']
+                continue
+            if k not in attr_dict:
+                # It's the first time we encounter this attribute
+                attr_dict[k] = fits.attrs[k]
+            elif k == 'ifo':
+                # We don't mind if this attribute is different
```

Comment:
note here also that the output file attrs will only be for one ifo even if more than one is input?

Reply:
I've added this into the comment, and also added a warning if files from multiple IFOs are used